src/device-detector.constants.ts

```diff
@@ -124,7 +124,7 @@ export const MOBILES_RE = {
 }
 
 export const TABLETS_RE = {
-  iPad: /iPad|iPad.*Mobile/,
+  iPad: /iPad|iPad.*Mobile|Macintosh.*Mac OS X/,
```

Comment:
@SaadQamar01 , this regex change will actually mark any device (desktop) as an iPad as well. Since we've added to the regex.

Reply:
Could you please test again.